src/Scene_Play.cpp

```diff
@@ -119,9 +126,12 @@ void Scene_Play::loadLevel(const std::string& path) {
 void Scene_Play::update() {
     m_currentFrame++;
     m_entities.update();
-    sMovement();
-    sCollision();
-    sAnimation();
+
+    if (!m_levelEnded) {
+        sMovement();
+        sCollision();
+        sAnimation();
+    }
```

Comment:
Will need to add future logic to play a sliding animation when m_levelEnded is true.